usmqe/api/graphiteapi/graphiteapi.py

```diff
@@ -89,17 +94,22 @@ def compare_data_mean(
                         expected_number_of_datapoints, len(graphite_data)))
             LOGGER.debug("mean of data from `{}` in Graphite: {}".format(
                 target, graphite_data_mean))
-            graphite_data_mean_sum += graphite_data_mean
+            if operation == 'sum' or graphite_data_mean_all == 0:
```

Comment:
It would be safer to really check what the first data value for subtraction is, without relying on never having 0 as a result of some step of the subtraction.